lectures/10.HPC_and_parallel_compute/scripts/multiprocessing_sequence_analysis.py

```diff
@@ -0,0 +1,68 @@
+import multiprocessing
```

Comment:
looks like this file is in here twice? Please delete one of them

Reply:
oh, i see! premature comment - however, you probably should write a custom util file and import the function

Reply:
Good point, util file implemented